src/test/java/seedu/guest/model/guest/NameTest.java

```diff
@@ -37,4 +39,11 @@ public void isValidName() {
         assertTrue(Name.isValidName("Capital Tan")); // with capital letters
         assertTrue(Name.isValidName("David Roger Jackson Ray Jr 2nd")); // long names
     }
+
+    @Test
+    public void hashcode() {
```

Comment:
Similarly, should we standardise the format for these tests with line spacing and comments?

Reply:
Agreed. Added line spacing and comments